covalent/_data_store/models.py

```diff
@@ -60,15 +60,30 @@ class Lattice(Base):
     # Name of the file containing the function string
     function_string_filename = Column(Text)
 
-    # Name of the file containing the serialized executor
-    executor_filename = Column(Text)
+    # Short name describing the executor ("local", "dask", etc)
+    executor = Column(Text)
```

Comment:
(non-blocking) Wouldn't `executor_name` be more clear?

Reply:
I agree that `executor_name` would be more descriptive. However, `executor` aligns with the corresponding electron metadata field name.